### DISCUSSION TITLE
Bug report about Lists when viewed via mobile browser  (Tested on Safari & Chrome for iOS)

### DISCUSSION THREAD

**Discussion by MussaCharles** [community/community#11091] [Other Feature Feedback, Questions, & Ideas]
Hi, thanks for this great feature. I have encountered  a minor UI bug as described below: - 

Currently when I open Github via Safari or Chrome on iPhone Lists which have longer subtitles causes text to overlap and display incorrect repositories count. This happens when device is in portrait mode. 

 Below are screenshots showing the issue for both chrome & safari.
  **Note:**  I have highlighted some of the parts with issues, Kindly refer to the screenshots. 
 
 
 **1. Chrome browser on iPhone**
 
![chrome](https://user-images.githubusercontent.com/25889447/152682468-ebffc75d-d1fa-4168-96ed-fc78431ab79a.jpg)

**2. Safari browser on iPhone**

![safari](https://user-images.githubusercontent.com/25889447/152682479-fc8a3cf9-e624-4489-a921-faf431375bd3.jpg)

**Comment by surapol2** [comment]
> Hi, thanks for this great feature. I have encountered a minor UI bug as described below: -
> 
> Currently when I open Github via Safari or Chrome on iPhone Lists which have longer subtitles causes text to overlap and display incorrect repositories count. This happens when device is in portrait mode.
> 
> Below are screenshots showing the issue for both chrome & safari. **Note:** I have highlighted some of the parts with issues, Kindly refer to the screenshots.
> 
> **1. Chrome browser on iPhone**
> 
> ![chrome](https://user-images.githubusercontent.com/25889447/152682468-ebffc75d-d1fa-4168-96ed-fc78431ab79a.jpg)
> 
> **2. Safari browser on iPhone**
> 
> ![safari](https://user-images.githubusercontent.com/25889447/152682479-fc8a3cf9-e624-4489-a921-faf431375bd3.jpg)
